sound/soc/sof/control.c

```diff
@@ -577,6 +577,13 @@ void snd_sof_control_notify(struct snd_sof_dev *sdev,
 	bool found = false;
 	int i, type;
 
+	if (cdata->type == SOF_CTRL_TYPE_VALUE_COMP_GET ||
+	    cdata->type == SOF_CTRL_TYPE_VALUE_COMP_SET) {
+		dev_err(sdev->dev,
+			"Component data is not supported in control notification\n");
+		return;
+	}
```

Comment:
We also have cases in other place in ipc.c and another in topology.c.

I thought to keep them, just in case if we have something emerging which would need it (I still can not figure out how it is supposed to wotk).

I can drop all of them in next version.